docs/versions.txt

```diff
@@ -13,7 +13,7 @@ JanusGraph.
 [options="header"]
 |==========================
 | JanusGraph | Cassandra | HBase | Bigtable | Elasticsearch | Solr | TinkerPop
-| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 2.z,5.z* |  5.2.z | 3.2.z |
+| 0.1.0 | 1.2.z, 2.0.z, 2.1.z | 0.98.z, 1.0.z, 1.1.z, 1.2.z | 0.9.z | 1.z*,2.z,5.z |  5.2.z | 3.2.z |
```

Comment:
No we dropped support for 1.z with previous update (#79) and this PR restores it. But unit tests require manually starting a 1.z ES instance so it'll be tough to maintain the support for new features/updates. Should we take 1.z back out of the table?